app/components/primary-button.tsx

```diff
@@ -0,0 +1,12 @@
+import { Button } from "@/components/ui/button";
+
+interface IButtonInfo {
+    onClick: (e: React.FormEvent) => void;
+    title: string;
+}
+
+export function PrimaryButton(props: IButtonInfo){
```

Comment:
Lint をかければ自動で修正できますが、TypeScript では arguments の丸括弧と statement の波括弧は間にスペースを入れますね

```suggestion
export function PrimaryButton(props: IButtonInfo) {
```